{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/forms/textarea.jinja

```diff
@@ -0,0 +1,21 @@
+{% from 'components/util.jinja' import attributes %}
+{% macro text_area(type="text", name=none, value=none, event_name="", color='primary', attrs={}, left_icon='', right_icon='') %}
+    <div x-data="input('update-{{ event_name | replace('_', '-') }}', '{{ value }}')"
+         class="w-full input">
+        <textarea
+            x-model.debounce.500ms="value"
+            {% set _ = attrs.update({"type": type, "name": name, "value": value}) %}
```

Comment:
might be a do statement?

Reply:
okay I can try it as a do statement. there are probably also 100 other places where "set"s could be converted to "do"s